src/interfaces.tsx

```diff
@@ -2,6 +2,11 @@ export interface Props {
   children: ChildFunction;
   query: Query;
   initialSize?: Size;
+  transformQuery: Function;
```

Comment:
Don't think we need to add `transformQuery` to `Props`, since `Props` is only used in `<ContainerQuery/>` API that can achieve similar functionality without `transformQuery`.